configuration_example-d2-docker.json

```diff
@@ -17,6 +18,65 @@
         "username": "system",
         "password": "System123"
     },
+    "departments":
+    {
+      "DEP1": {
+        "datasets": [
+          "example_uid", "example_uid2"
+        ]
+      },
+      "DEP2": {
+        "eventprograms": [
+          "example_uid"
+        ],
+        "datasets": [
+          "example_uid"
+        ]
+      },
+      "DEP3": {
+        "eventprograms": [
+          "example_uid"
+        ],
+        "datasets": [
+          "example_uid"
+        ],
+        "trackerprograms": [
+          "example_uid"
+        ]
+      }
+    },
+    "preprocess": [
+    {
+      "selectDepartament": "DEP1",
+      "selectMDType": ["datasets"],
+      "action": "removeData"
+    },
+    {
+      "selectDepartament": "DEP2",
+      "selectMDType": ["datasets","eventPrograms"],
+      "action": "removeData"
+    },
+    {
+      "selectDepartament": "DEP3",
+      "selectMDType": ["datasets"],
+      "selectDatasets": [
+        "HtZb6Cg7TXo", "Tu81BTLUuCT", "Wxl8hxuOsIQ", "m5MiTPdlK17", "pZ3XRBi9gYE"
+      ],
+      "action": "removeData"
+    },
+    {
+      "selectDepartament": "DEP3",
+      "selectMDType": ["eventPrograms"],
+      "selectOrgUnits": ["example_uid"],
```

Comment:
[comment] we typically used names for users, user names, and so on; in this new code we start using IDs for datasets, organisation units, programs, ..., will be that a problem or is it ok?

Reply:
I have changed all the uids fields by uids to make more evident the difference between actions and codes.
Do you think this could be a problem?